libcloudforensics/scripts/ebs_snapshot_copy_aws.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -x
+
+set -o pipefail
+
+snapshot=%s
+bucket=%s
+
+function ebsCopy {
+	# params
+	snapshot=$1
+	bucket=$2
+
+	# Get details about self
+	region=$(curl -s http://169.254.169.254/latest/meta-data/placement/region)
+	az=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
+	instance=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
+
+	# create the new volume
+	volume=$(aws ec2 --region $region create-volume --availability-zone $az --snapshot-id $snapshot --tag-specification 'ResourceType=volume,Tags=[{Key=Name,Value=volumeToCopy}]' | jq -r .VolumeId)
```

Comment:
We already have functionality to create volume copies / instances and to attache volume copies. Perhaps the functionality here would be better represented in Python through the library, and the script could limit itself to performing `dd`. Would be easier to catch errors.

@tomchop wdyt?

Reply:
As discussed OOB - Short version is other methods presented such problems that made this the best method I could come up with.